src/components/input/input.js

```diff
@@ -325,7 +328,9 @@ function placeholderDirective() {
     var placeholderText = attr.placeholder;
     element.removeAttr('placeholder');
 
-    inputContainer.element.append('<div class="md-placeholder">' + placeholderText + '</div>');
+    var placeholder = '<div class="md-placeholder" ng-click="delegateClick()">' +
```

Comment:
What happens if `placeholderText == undefined`? 
What is the sizing of the `md-placeholder`.
Is usable for all our input demos on browsers ?